docs & test: add README and basic AirdropNFT test

tesnetra opened the pull request:
- Added a basic test for the AirdropNFT contract to ensure successful deployment
- Created a clear and simple README for onboarding contributors
- Removed default Lock.ts test from Hardhat boilerplate

AndresRamirez9912 commented:
Hello @tesnetra 
I'm interested in your PR, but I want to check if this is all? or you missed pushing the commit with more instructions on the README.md? 
![image](https://github.com/user-attachments/assets/0692ad83-b859-4a72-86ae-c19769548bf3)

tesnetra commented:
Hi @AndresRamirez9912,
Thank you! Yes, I’ve pushed the latest version of the README.md with full
setup instructions and clarification regarding the removed boilerplate.
Apologies for the confusion earlier — I had forgotten to push the file
initially.
Let me know if anything else is needed! 🙌

Pada Rab, 2 Jul 2025 pukul 22.21 Andres Ramirez ***@***.***>
menulis:

> *AndresRamirez9912* left a comment (KiiChain/kii-solidity-contracts#3)
> <https://github.com/KiiChain/kii-solidity-contracts/pull/3#issuecomment-3028069055>
>
> Hello @tesnetra <https://github.com/tesnetra>
> I'm interested in your PR, but I want to check if this is all? or you
> missed pushing the commit with more instructions on the README.md?
> image.png (view on web)
> <https://github.com/user-attachments/assets/0692ad83-b859-4a72-86ae-c19769548bf3>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/KiiChain/kii-solidity-contracts/pull/3#issuecomment-3028069055>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANQ5AZ4FC2CYLV63TGYXPE33GPTG3AVCNFSM6AAAAACAPOMRL6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTAMRYGA3DSMBVGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

AndresRamirez9912 commented on README.md:
```diff
@@ -1,5 +1,63 @@
-# sKII Internal Swap Contract
```
Please change "sKii" to "Kii"

AndresRamirez9912 commented on README.md:
````diff
@@ -1,5 +1,63 @@
+### 1. Clone the Repository
+
+```bash
+git clone https://github.com/tesnetra/kii-solidity-contracts.git
````
Please add the link pointing to our Kiichain repo

AndresRamirez9912 commented:
@tesnetra Thanks for pushing the complete changes. I have left some comments. I will be looking forward your commit

tesnetra commented:
Hi @AndresRamirez9912,
Thank you again! I've updated the README.md as requested:

   - Replaced “sKii” with “Kii”
   - Added an intro paragraph about the swap contract
   - Included the main repo link

Let me know if there's anything else to improve.

Pada Kam, 3 Jul 2025 pukul 07.03 Andres Ramirez ***@***.***>
menulis:

> ***@***.**** approved this pull request.
> ------------------------------
>
> In README.md
> <https://github.com/KiiChain/kii-solidity-contracts/pull/3#discussion_r2181121229>
> :
>
> > @@ -1,5 +1,63 @@
> -# sKII Internal Swap Contract
>
> Please change "sKii" to "Kii"
> ------------------------------
>
> In README.md
> <https://github.com/KiiChain/kii-solidity-contracts/pull/3#discussion_r2181121592>
> :
>
> >
> -This is a simple swap contract that utilizes the cosmos bank module on the Kii EVM testnet.  It is meant to swap EVM kii tokens with internal cosmos sKii tokens and vice versa.
> +## 🚀 Getting Started
> +
> +Follow these steps to set up the development environment and test the contracts locally.
> +
> +### 1. Clone the Repository
> +
> +```bash
> +git clone https://github.com/tesnetra/kii-solidity-contracts.git
>
> Please add the link pointing to our Kiichain repo
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/KiiChain/kii-solidity-contracts/pull/3#pullrequestreview-2980870140>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANQ5AZ7XAC5PAUSWZKNFI233GRQMBAVCNFSM6AAAAACAPOMRL6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDSOBQHA3TAMJUGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

AndresRamirez9912 commented on README.md:
````diff
@@ -1,5 +1,73 @@
+### 1. Clone the Repository
+
+```bash
+git clone https://github.com/tesnetra/kii-solidity-contracts.git
````
This one should be
`
git clone https://github.com/KiiChain/kii-solidity-contracts
`

AndresRamirez9912 commented:
@tesnetra Hello, check the section I highlighted to you, which refers to your repo. Please change that to refer to our repo.
